sphinx_gallery/docs_resolv.py

```diff
@@ -208,8 +208,11 @@ def _get_index_match(self, first, second):
                     return None
         return match
 
-    def _get_link_type(self, cobj):
+    def _get_link_type(self, cobj, use_full_module=False):
```

Comment:
I think with that option we would have it save for older versions because it defaults to use the `module_short` key.